integration_tests/tests/forseti/controls/inventory.rb

```diff
@@ -0,0 +1,95 @@
+require 'json'
+control 'inventory' do
+
+    describe "Inventory" do
+        before :context do
+            command("forseti inventory purge 0").result
+            command("forseti inventory create").result
+            command("sudo apt-get -y install mysql-client").result
+
+             # This variable cannot be used after all the inventories have been purged..
+            @inventory_id = JSON.parse(command("forseti inventory list").stdout).fetch("id")
+        end
+
+        describe "Create an inventory" do
+
+            it "should be visible from the command-line" do
+                expect(command("forseti inventory list").stdout).to match /SUCCESS/
+            end
+
+            it "should be visible in the database" do
+                expect(command("mysql -u root --host 127.0.0.1 --database forseti_security --execute \"select COUNT(DISTINCT gcp_inventory.inventory_index_id) from gcp_inventory join inventory_index ON inventory_index.id = gcp_inventory.inventory_index_id;\"").stdout).to match /1/
+            end
+
+             # Below assertion should be uncommented after populating test data.
+             # it "should be visible in the database" do
+             #    expect(command("mysql -u root --host 127.0.0.1 --database forseti_security --execute \"select count(DISTINCT resource_id) from gcp_inventory where category='resource' and resource_type = 'project';\"").stdout).to match /1/
```

Comment:
Before we merge, I like to see that we can assert the actual result in the database, against the test resources that were defined to be populated.  i.e. if we define N projects to be created in the test environment, I want to ensure the N projects are in the inventory database.

Reply:
I have uncommented this check. Current count of the projects in the database is high at this time as project was created for test purpose every time; I have removed that logic. A lot of projects are in the pending delete state at this moment and so they are reflected when the query in question is executed. It is likely that the test will fail when the projects are deleted. The count needs to be updated at that time.

Reply:
Deleted projects should be able to be filtered out by the lifecycle state.

Even beyond projects, this would be very useful other resources.  e.g. if i defined 10 buckets to be created in my test inv, is there really 10 buckets in the database.

Reply:
Done